src/vectorchord/getting-started/installation.md

```diff
@@ -0,0 +1,148 @@
+
+1. Download the deb package in [the release page](https://github.com/tensorchord/vectorchord/releases/latest), and type `sudo apt install vchord-pg15-*.deb` to install the deb package.
+
+2. Configure your PostgreSQL by modifying the `shared_preload_libraries` and `search_path` to include the extension.
+
+```sh
+psql -U postgres -c 'ALTER SYSTEM SET shared_preload_libraries = "vchord.so"'
+psql -U postgres -c 'ALTER SYSTEM SET search_path TO "$user", public, vchord'
+# You need restart the PostgreSQL cluster to take effects.
+sudo systemctl restart postgresql.service   # for vectorchord running with systemd
+```
+
+3. Connect to the database and enable the extension.
+
+```sql
+DROP EXTENSION IF EXISTS vchord;
+CREATE EXTENSION vchord CASCADE;
+```
+
+## From ZIP package
+
+::: tip
+Installation from the ZIP package requires a dependency on `GLIBC >= 2.31`, e.g:
+- `RHEL 9` or later
+:::
+
+For systems that are not Debian based and cannot run a Docker container, please follow these steps to install:
+
+1. Before install, make sure that you have the necessary packages installed, including `PostgreSQL`, `pg_config`, `unzip`, `wget`.
+
+```sh
+# Example for RHEL 9 dnf
+# Please check your package manager
+sudo dnf install -y unzip wget libpq-devel
+sudo dnf module install -y postgresql:15/server
+sudo postgresql-setup --initdb
+sudo systemctl start postgresql.service
+sudo systemctl enable postgresql.service
+```
+
+2. Verify whether `$pkglibdir` and `$shardir` have been set by PostgreSQL. 
+
+```sh
+pg_config --pkglibdir
+# Print something similar to:
+# /usr/lib/postgresql/15/lib or
+# /usr/lib64/pgsql
+
+pg_config --sharedir
+# Print something similar to:
+# /usr/share/postgresql/15 or
+# /usr/share/pgsql
+```
+
+3. Download the zip package in [the release page](https://github.com/tensorchord/vectorchord/releases/latest) and extract it to a temporary directory.
+
+```sh
+wget https://github.com/tensorchord/VectorChord/releases/download/0.1.0/vchord-pg15_x86_64-unknown-linux-gnu_0.1.0.zip -O vchord.zip
+unzip vchord.zip -d vchord
+```
+
+4. Copy the extension files to the PostgreSQL directory.
+
+```sh
+# Copy library to `$pkglibdir`
+sudo cp vchord/vchord.so $(pg_config --pkglibdir)/
+# Copy schema to `$shardir`
+sudo cp vchord/vchord--* $(pg_config --sharedir)/extension/
+sudo cp vchord/vchord.control $(pg_config --sharedir)/extension/
+```
+
+5. Configure your PostgreSQL by modifying the `shared_preload_libraries` and `search_path` to include the extension.
+
+```sh
+psql -U postgres -c 'ALTER SYSTEM SET shared_preload_libraries = "vchord.so"'
+psql -U postgres -c 'ALTER SYSTEM SET search_path TO "$user", public, vchord'
+# You need restart the PostgreSQL cluster to take effects.
+sudo systemctl restart postgresql.service   # for vectorchord running with systemd
+```
+
+6. Connect to the database and enable the extension.
+
+```sql
+DROP EXTENSION IF EXISTS vchord;
+CREATE EXTENSION vchord CASCADE;
+```
+
+## From source
+
+TODO
```

Comment:
How to build from source? Could I refer to here?

https://github.com/tensorchord/VectorChord/tree/main/scripts#build-docker cc @kemingy

Reply:
I think https://github.com/tensorchord/VectorChord/blob/45c573b6ee890193c4d0d6f3e64f538a15d54494/.github/workflows/psql.yml#L76-L79 is much easier unless you already have the pgrx & postgres-dev env.

BTW, we build on Ubuntu 22.04.